README.adoc

```diff
@@ -345,23 +345,22 @@ Tests run: 2, Failures: 0, Errors: 0, Skipped: 0
 
 Deploying the application to a Kubernetes environment using the Open Liberty Operator is an optional step in this guide.
 
-
-// =================================================================================================
-// Deploying the application in the prod environment with Kubernetes
-// =================================================================================================
-=== Deploying the application in the prod environment with Kubernetes
-
 To further explore deploying microservices using Kubernetes and the Open Liberty Operator, you can read the following guides:
 
 * https://openliberty.io/guides/openliberty-operator-intro.html[Deploying a microservice to Kubernetes using Open Liberty Operator^]
 * https://openliberty.io/guides/openliberty-operator-openshift.html[Deploying a microservice to OpenShift 4 using Open Liberty Operator^]
 
+Ensuring security in the production environment is essential. In the previous sections, you have learnt how to externalize the credentials and other properties for accessing the `system` service by using the MicroProfile Config that makes the application more adaptable to different environments without changing code or configuiration files.  In the this section, you'll learn how to make use of Kubernetes secret to provide the credentials and how to pass them to the `query` service through the MicroProfile Config.
```

Comment:
```suggestion
Ensuring security in the production environment is essential. In the previous sections, you have learnt how to externalize credentials and other properties for accessing the `system` service by using MicroProfile Config, making the application more adaptable to different environments without changing code or configuration files. In the this section, you'll learn how to make use of Kubernetes secrets to provide the credentials and how to pass them to the `query` service through the MicroProfile Config.
```

I think "In the previous sections, you have learnt how to externalize the credentials and other properties for accessing the `system` service by using the MicroProfile Config that makes the application more adaptable to different environments without changing code or configuiration files." is too lengthy? so I split it with a comma. 

Also, 'configuration' was spelled incorrectly in the original sentence.